source/features/toggle-files-button.css

```diff
@@ -8,8 +8,9 @@
 	border-radius: 3px;
 }
 
-.repository-content:not(.rgh-files-hidden) .commit-tease .octicon-chevron-down {
-	transform: rotate(0.5turn);
+.repository-content:not(.rgh-files-hidden) .commit-tease .octicon-unfold,
+.repository-content.rgh-files-hidden .commit-tease .octicon-fold {
```

Comment:
Could you check how GitHub does this on comments? Maybe all we need is a class, without extra CSS

Reply:
It seems to be specific to the `<details>` element:

![Screenshot_2020-06-06_18-39-06](https://user-images.githubusercontent.com/202916/83949591-0a66ff80-a825-11ea-8019-aebe2c5edd8b.png)

Reply:
To be fair, this whole feature can probably be handled by GitHub, meaning technically we only need to add those `.Detail` classes (which aren't specific to `detail` element) and skip the delegation too. But if you're not interested, no problem, it's just a possible improvement

Reply:
I'm not sure if this will work because I think the functionality is coupled to the actual `<details>` / `<summary>` elements and the classes are only there for styling.

Either way, I think this should be part of a separate PR.

Reply:
> I'm not sure if this will work because I think the functionality is coupled to the actual `<details>` / `<summary>` elements and the classes are only there for styling.

Nope, GitHub has this:


```css
.Details--on .Details-content--shown,
.Details:not(.Details--on) .Details-content--hidden {
 display:none!important
}
.Details:not(.Details--on) .Details-content--hidden-not-important {
 display:none
}
```

```js
const n = null != e ? e : !o.classList.contains("open");
o.classList.toggle("open", n),
o.classList.toggle("Details--on", n),
```

> Either way, I think this should be part of a separate PR.

👍